packages/sonarwhal/src/lib/utils/resource-loader.ts

```diff
@@ -234,8 +234,8 @@ export const loadResource = (name: string, type: ResourceType, configurations: A
     const currentProcessDir = process.cwd();
 
     const sources: Array<string> = [
-        `@sonarwhal/${key}`, // Officially supported package
-        `sonarwhal-${key}`, // Third party package
+        `@sonarwhal/${type}-${packageName}`, // Officially supported package
```

Comment:
shouldn't we update the value of `key` for this?

Reply:
I don't think so, we are using the key to save in the cache. But in the cache I want to save the rule it self (i.e. `rule-typescript-config/isvalid`) but I need to load the package that contains the rule (i.e. `@sonarwhal/rule-typescript-config`).

Reply:
Now it makes more sense, can you add a comment explaining exactly this?

Reply:
sure!